src/style.css

```diff
@@ -1 +1,52 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.logo {
+  position: absolute;
+  left: 50px;
```

Comment:
use horizontal padding for header (and it will 50px from the left and right edges of the viewport)

Reply:
and do not use absolute here